package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "mx-sdk-js-bridge",
+  "version": "1.0.0",
```

Comment:
Can also be pre `v1`. E.g. `0.1.0`. If you commit to `v1` right now, be careful with upcoming breaking changes.

Reply:
done